hledger-web/Hledger/Web/WebOptions.hs

```diff
@@ -47,7 +48,7 @@ webflags =
       ["socket"]
       (\s opts -> Right $ setopt "socket" s opts)
       "SOCKET"
-      "use the given socket instead of the given IP and port (implies --serve)"
+      "use the given socket instead of the given IP and port (implies --serve, ignored it not available)"
   , flagReq
```

Comment:
it not available -> if not supported by OS ?

But actually, if it's not supported and they try to use it, we should give an error rather than silently running in a different mode.

Reply:
That would make more sense. I'll add an error.